src/pages/contact/components/socialMedia.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import {
+  faDiscord, faFacebook, faInstagram, faLinkedin, faSlack, faTwitter,
+} from '@fortawesome/free-brands-svg-icons';
+import Container from '@material-ui/core/Container';
+import { FontAwesomeLink } from '../../../components/icons';
+
+function SocialMedia() {
+  return (
+    <Container>
+      <h2 style={styles.header}>Follow us on social media</h2>
+      <Grid
+        container
+        direction="row"
+        justify="space-evenly"
+        alignItems="center"
+        spacing={2}
+      >
+        <Grid item xs={12} sm={6} md={4} lg={2} style={styles.grid}>
```

Comment:
I think these would look better closer together
![image](https://user-images.githubusercontent.com/35578281/123249415-ecfbe680-d4b6-11eb-9f0a-87a78a8d8a24.png)

Also please fix up formatting on mobile
![image](https://user-images.githubusercontent.com/35578281/123249511-06049780-d4b7-11eb-955c-89aaa1e1a592.png)

Reply:
I think you are on an older version of the branch